### PR TITLE
[WIP] Add remove-members helper script

### PR DESCRIPTION
**Pull request by mrbobbytables**
This is the script used to generate #2047 
It needs some final polish like usage information, but figured I'd share for initial review / vetting alongside #2047

/hold

### CODE REVIEW

File: hack/remove-members.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly CONFIG_PATH="$REPO_ROOT/config"
+readonly DRYRUN="${DRYRUN:-true}"
```
Comment (nikhita) [inline hack/remove-members.sh:22]: Can we make this a flag?

File: hack/remove-members.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly CONFIG_PATH="$REPO_ROOT/config"
```
Comment (nikhita) [inline hack/remove-members.sh:21]: Can/should we make this configurable? In cases where someone is active in `kubernetes-sigs` but not active in `kubernetes`, or if we just want to remove members from `kubernetes`?

File: hack/remove-members.sh (hunk 1 above)
Comment (mrbobbytables) [reply hack/remove-members.sh:22]: Yeah, this was me being kind of lazy to get it done lol

File: hack/remove-members.sh (hunk 2 above)
Comment (mrbobbytables) [reply hack/remove-members.sh:21]: I can do that 👍

**Comment by mrbobbytables** [comment]
There is one issue I just thought of, it currently won't remove members if they have a comment after their name like
` - foo # herp derp`
Lucked out with the original batch

**Comment by fejta** [comment]
Thoughts on putting this somewhere aside from hack? The unfortunately named hack folder generally seems to be used for lint-type updates and/or verifications. 

Isn't this an operational thing? AKA I want to remove Erick from the org and this is the tool to do so, which just happens to be in bash right now.

**Comment by justaugustus** [comment]
Adding on to @fejta's, I think this is important enough and of sufficient complexity that it should be written in Go.

**Comment by mrbobbytables** [comment]
I'm okay with that -- I'm a little low on time at the moment, but I'll see what I can do to get something a bit better pushed up. Either way will get something before the next round of clean up. 👍 

/close

**Comment by nikhita** [comment]
Created https://github.com/kubernetes/org/issues/2052 so that we don't forget this. I'm also happy to take a look at it later :)

**Pull request closed without merging**